laalaa/apps/advisers/management/commands/worker_health_checks.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import sys
+import requests
+import logging
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+from advisers.models import Organisation, Import, IMPORT_STATUSES
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Check access to the database, celery workers and postcodes.io"
+    import_created_status_stuck_interval = datetime.timedelta(minutes=10)
+    import_running_status_stuck_interval = datetime.timedelta(minutes=60)
+
+    def handle(self, *args, **options):
+
+        try:
+            self.check_database_access()
+            self.check_import_stuck_in_progress()
+            self.check_celery_worker()
+            self.check_postcodes_io()
+        except Exception as error:
+            print(str(error))
+            sys.exit(1)
+
+    def check_postcodes_io(self):
+        response = requests.get(f"{settings.POSTCODES_IO_URL}/postcodes/SW1A1AA", timeout=5)
+        if response.status_code != 200:
+            raise CommandError(f"postcodes.io check has failed. Expected 200 response got {response.status_code}")
+
+    def check_database_access(self):
+        try:
+            # ability to connect to the db; doesn't matter if the result is True or False
+            Organisation.objects.exists()
+        except Exception:
+            raise CommandError("database access check has failed.")
+
+    def check_import_stuck_in_progress(self):
+        last_import = Import.objects.last()
+        if not last_import.is_in_progress():
```

Comment:
I think this could be `if not last_import or not last_import.is_in_progress()` to handle the case of no imports in the db

Reply:
Updated. Good spot, thank you